hypervisor/scenarios/hybrid/vm_configurations.c

```diff
@@ -13,8 +13,7 @@ struct acrn_vm_config vm_configs[CONFIG_MAX_VM_NUM] = {
 		CONFIG_SAFETY_VM(1),
 		.name = "ACRN PRE-LAUNCHED VM0",
 		.guest_flags = 0UL,
-		.vcpu_num = 1U,
-		.vcpu_affinity = VM0_CONFIG_VCPU_AFFINITY,
+		.pcpu_affinity_bitmap = VM0_CONFIG_PCPU_BITMAP,
```

Comment:
Given we call it pcpu_affinity_bitmap, the MACRO name can still be VM0_CONFIG_VCPU_AFFINITY. Though within VM0_CONFIG_VCPU_AFFINITY, we can use PCPU_BITMAP. How do you think?

Reply:
pcpu_xxx == vcpu_xxx looks a bit weird.

Compared with "vCPU affinity", it seems "CPU affinity" is a more common term.  Simply change "vcpu_affinity" to "cpu_affinity" seems solve the problem. It's short and seems accurate.

Changes in vm_configuration.c:
-.vcpu_affinity = VMx_CONFIG_VCPU_AFFINITY,
+.cpu_affinity_bitmap = VMx_CONFIG_CPU_AFFINITY,

Changes in vm_configuration.h:
-#define  VMx_CONFIG_VCPU_AFFINITY {AFFINITY_CPU(xU), AFFINITY_CPU(xU)}
+#define  VMx_CONFIG_CPU_AFFINITY   (AFFINITY_CPU(xU) | AFFINITY_CPU(xU))

Changes in scenario xml files. And update comments.
-vcpu_affinity
+cpu_affinity

Reference:
Wiki: Processor affinity, or CPU pinning or "cache affinity", enables the binding and unbinding of a process or a thread to a central processing unit (CPU) or a range of CPUs.

VMWare: By specifying a CPU affinity setting for each virtual machine, you can restrict the assignment of virtual machines to a subset of the available processors in multiprocessor systems.